RadarSDK/Include/Radar.h

```diff
@@ -765,6 +774,34 @@ typedef void (^_Nonnull RadarRouteMatrixCompletionHandler)(RadarStatus status, R
                        units:(RadarRouteUnits)units
            completionHandler:(RadarRouteMatrixCompletionHandler)completionHandler NS_SWIFT_NAME(getMatrix(origins:destinations:mode:units:completionHandler:));
 
+/**
+ Sends a custom event.
+
+ @param customType The user-defined type of the event.
+ @param metadata The metadata associated with the event.
+ @param completionHandler A completion handler.
+
+ @see https://radar.com/documentation/api#send-a-custom-event
+ */
++ (void)sendEvent:(NSString *)customType
```

Comment:
do you think `customType` is a better named parameter than `name`? `event.customType` can be an internal impl detail and doesn't necessarily need to be indicated to the caller

I'm also fine if you believe strongly that `customType` is better - at the end of the day, it is overwriting `event.type`

Reply:
That's what @nickpatrick suggested, and I'm ok with it.